pkg/webreg/webreg.go

```diff
@@ -328,6 +333,195 @@ const templateDefinitions = `
 {{ end }}
 `
 
+const helpPage = `
+<h1><a href="/">Openshift CI Step Registry</a></h1>
+<h1>What is the Multi-Stage Test and the Test Step Registry?</h1>
+
+<p>
+The multistage test style in the ci-operator is a modular test design that
+allows users to create new tests by combining smaller, individual test steps.
+These individual tests can be put into a shared registry that other tests can
+access. This results in test workflows that are easier to maintain and
+upgrade as multiple test workflows can share steps and don’t have to each be
+updated individually to fix bugs or add new features. It also reduces the
+chances of a mistake when copying a feature from one test workflow to
+another.
+</p>
+
+<p>
+To understand how the multistage tests and registry work, we must first talk
+about the 3 components of the test registry and how to use those components
+to create a test.
+</p>
```

Comment:
I think we should have a three-item bullet list here with a one-sentence summary for each item, before we go to the gory details of individual items.

- Reference: A reference is the lowest level component in the test step registry, describes an individual test step.
- Chain: A chain is a registry component that specifies multiple steps to be run. Any item of the chain can be either a reference or another chain.
- Workflow: A workflow is the highest level component of the step registry. It contains three chains: pre, test, post.

Or something similar.